src/Twig/TwigExtension.php

```diff
@@ -46,18 +48,35 @@ protected function getTemplateFilenameIterator(): \Traversable
     }
 
     /**
+     * Add a directory to the Twig loader.
+     *
+     * @param string $namespace the twig namespace the path should be added to
+     * @param string $path      the template directory to add
+     *
+     * @return $this
+     */
+    public function addTwigPath(string $namespace, string $path): ExtensionInterface
```

Comment:
Should this live in [AbstractTwigExtension](https://github.com/LastCallMedia/Mannequin/blob/master/src/Twig/AbstractTwigExtension.php) instead, so that it's also available to [TwigExtension](https://github.com/LastCallMedia/Mannequin/blob/master/src/Twig/TwigExtension.php)?

Reply:
Good question.  We added it both the TwigExtension and the DrupalExtension individually.  I'm thinking of it right now as an implementation detail of these two extensions, not of any extension that deals with Twig at all.  Namespaces are actually only an implementation detail of the `\Twig_Loader_Filesystem`, so I don't think it makes sense to require that any extension that deals with Twig also knows how to deal with namespaces.

Reply:
Ah, right on! Nice stuff, thanks.